### ISSUE TITLE
Strange issue with Twilight Forest and Carry On combined

### ISSUE THREAD

**Issue by Sunconure11** [Tschipp/CarryOn#63]
### Expected Behavior

I should not have ice blocks surrounding me as if I was hit by an ice bow

### Actual Behavior

![screenshot 2017-12-12 12 06 00](https://user-images.githubusercontent.com/5614902/33898011-144223d8-df35-11e7-9bb1-bfd8a4c0ade2.png)

### Steps to Reproduce

1. Install Twilight Forest + Carry On
2. Pick up anything.

### Version of Minecraft, Carry On, Forge

1.12.2 minecraft
14.23.1.2562 forge
v1.7 carry on

### Notes:

Might want to add an issue to the twilight forest repo if this isn't your end. https://github.com/TeamTwilight/twilightforest

**Comment by Tschipp** [comment]
Call me crazy, but I think this is a feature of Twilight Forest that adds these Ice blocks if you have slowness.

**Comment by Sunconure11** [comment]
Alright, I'll hit myself with a slowness potion.

**Comment by Sunconure11** [comment]
This issue appears to have been resolved.

**Comment by Tschipp** [comment]
Great!

**Comment by capnkirok** [comment]
Yay!

**Comment by ExcitedName** [comment]
Unfortunately the problem persists. It is sporadic however. This is my first modpack for 1.12.2. So I am not sure if it is specifically related to Twilight Forest.
![2018-02-21_22 18 03](https://user-images.githubusercontent.com/17900146/36523390-2f98b954-1756-11e8-944c-bb3e228f64bf.png)

Minecraft 1.12.2
forge-1.12.2-14.23.2.2611-universal
CarryOn+MC1.12.2+v1.7.2
twilightforest-1.12.2-3.6.345-universal

Would you like me to create a new issue for this or just leave it here?

**Comment by Tschipp** [comment]
It is definitely related to twilight forest... reopening.

**Comment by capnkirok** [comment]
Should we pull in Drullkus?

**Comment by Tschipp** [comment]
I mean... I am still failing to recreate it reliably... so I would say not until we know more of how to recreate this.

**Comment by Drullkus** [comment]
I know what's happening. It's because the player has slowness and particles are turned off for the effect. TF marks this as player being frosted or whatever. Kind of a weird implementation, I could solve this by adding a new potion effect on our end. https://github.com/TeamTwilight/twilightforest/blob/61b418b9bc205ac2ce87a6c540ba4a51ad0e4f0d/src/main/java/twilightforest/client/TFClientEvents.java#L54-L64

**Comment by Tschipp** [comment]
You could do that with a custom potion effect, but you can also just add a custom tag or something to the player. Would save you some work

**Comment by Tschipp** [comment]
This appears to have been fixed by Twilight Forest adding a new effect for the ice slowness
